### DISCUSSION TITLE
Shadows not following the animation in synchro

### DISCUSSION THREAD

**Discussion by hinchmonster** [iTwin/itwinjs-core#4639] [Q&A]
https://github.com/iTwin/itwinjs-core/discussions/4597#discussioncomment-4109435
As followed on from the discussion on the thread linked above:

also something not sure if it is an unreal issue of the datasmith issues the shadows from the animation are always there even before the object has appeared is this a me issue or an datasmith issue?

Reply:

see this section of our webinar on the iTwin Exporter for Datasmith: https://youtu.be/IV0eBECws1k?t=2510
You want to disable "cast ray traced shadows" on your transparent materials.

 --------------------------------------------------------------

@MichaelBelousov I followed that tutorial (https://www.youtube.com/watch?v=IV0eBECws1k ) and it works perfectly for all transparent materials but for masked materials i.e concrete and steel it doesnt work.

**Comment by MichaelBelousov** [comment]
Hmm as you should be able to see in that video, plenty of masked geometry is not casting shadows https://youtu.be/IV0eBECws1k?t=2727
Is all masked geometry casting shadows for you? Are you on Unreal Engine 5.0.0?

**Reply by hinchmonster** [reply]
I am using 5.0.2 and it is not showing the same. Even before any materials are applied the shadows are there constantly.

**Reply by MichaelBelousov** [reply]
you should be using fully dynamic lighting, are you?

**Reply by hinchmonster** [reply]
i have used whatever lighting is standard in basic arch vis template

**Reply by MichaelBelousov** [reply]
That should be dynamic. Can you find the lights in your scene in the scene outliner (in the top right by default) and in their properties make sure they are all set to dynamic? not static or movable)
